### PR TITLE
updated Network.js

### PR DESCRIPTION
**Pull request by arjunvijayvargiya**
Reindented

### CODE REVIEW

**Comment by arjunvijayvargiya** [comment]
Hi @smcgregor ,
                              Please see to this.

**Comment by vshivam** [comment]
@smcgregor Yep. The method already exists in the JavaScript Bridge.

File: shared/javascripts/network_service.js [hunk 1]
```diff
@@ -437,15 +437,19 @@ var privlyNetworkService = {
...
+      androidJsBridge.showLoginActivity();
+    } else {
+      // Don't show the nav at all if the content is injected.
+      if (typeof privlyHostPage !== "undefined" && privlyHostPage.isInjected()) {
+        return;
+      }
+      $(".logged_in_nav").hide();
+      $(".logged_out_nav").show();
+      $(".injected_hide").show();
+      privlyNetworkService.mobileHide();
```
Comment (smcgregor) [inline shared/javascripts/network_service.js]: You can move this into the mobile block you defined since it only applies to mobile.

**Comment by smcgregor** [comment]
I'll make a few modifications and merge.

**Pull request merged**